td-layout/src/build_time.rs

```diff
@@ -59,6 +60,4 @@ pub const TD_SHIM_TEMP_HEAP_BASE: u32 = 0xFF062000; // TD_SHIM_FIRMWARE_BASE + T
 pub const TD_SHIM_PAYLOAD_BASE: u32 = 0xFF082000; // TD_SHIM_FIRMWARE_BASE + TD_SHIM_PAYLOAD_OFFSET
 pub const TD_SHIM_IPL_BASE: u32 = 0xFFCB0000; // TD_SHIM_FIRMWARE_BASE + TD_SHIM_IPL_OFFSET
 pub const TD_SHIM_RESET_VECTOR_BASE: u32 = 0xFFFF8000; // TD_SHIM_FIRMWARE_BASE + TD_SHIM_RESET_VECTOR_OFFSET
-pub const TD_SHIM_RESET_SEC_CORE_ENTRY_POINT_ADDR: u32 = 0xFFFFFFD4; // 0xFFFFFFFF - 0x20 - 12 + 1
-pub const TD_SHIM_RESET_SEC_CORE_BASE_ADDR: u32 = 0xFFFFFFD8; // 0xFFFFFFFF - 0x20 - 8 + 1
-pub const TD_SHIM_RESET_SEC_CORE_SIZE_ADDR: u32 = 0xFFFFFFDC; // 0xFFFFFFFF - 0x20 - 4 + 1
+pub const TD_SHIM_SEC_CORE_INFO_BASE: u32 = 0xFFFFFFAC; // 0xFFFFFFFF - TD_SHIM_SEC_INFO_OFFSET + 1
```

Comment:
why we need this change?

Reply:
Just split these changes to a separate commit